src/components/SettingsPage/sections/Experimental.js

```diff
@@ -0,0 +1,88 @@
+// @flow
+
+import React, { PureComponent } from 'react'
+import { translate, Trans } from 'react-i18next'
+import { setEnvUnsafe, isEnvDefault } from '@ledgerhq/live-common/lib/env'
+import { experimentalFeatures, isReadOnlyEnv } from 'helpers/experimental'
+import type { T } from 'types/common'
+import TrackPage from 'analytics/TrackPage'
+import IconAtom from 'icons/Atom'
+import IconSensitiveOperationShield from 'icons/SensitiveOperationShield'
+import Disclaimer from 'components/Disclaimer'
+
+import {
+  SettingsSection as Section,
+  SettingsSectionHeader as Header,
+  SettingsSectionBody as Body,
+  SettingsSectionRow as Row,
+} from '../SettingsSection'
+import ExperimentalSwitch from '../ExperimentalSwitch'
+
+type Props = {
+  t: T,
+}
+
+type Feature = {
+  type: string,
+  name: string,
+  title: string,
+  description: string,
+  valueOn?: any,
+  valueOff?: any,
+}
+
+const ExperimentalFeatureRow = ({ feature }: { feature: Feature }) => {
+  const { type, ...rest } = feature
+  let children = null
+
+  if (type === 'toggle') {
```

Comment:
instead, define a map `controls = { toggle: ExperimentalSwitch }` . (if we have more cases in future it won't be a big series of if)